types/jspreadsheet-ce/index.d.ts

```diff
@@ -0,0 +1,915 @@
+// Type definitions for jspreadsheet-ce 4.7
+// Project: https://github.com/jspreadsheet/ce
+// Definitions by: arx-8 <https://github.com/arx-8>
```

Comment:
It's somewhat unusual to add an owner other than yourself.  Was this intentional?

Reply:
These are originally created by arx-8 for jexcel. They have renamed the library to jspreadsheet-ce. I only copied types from jexcel to jspreadsheet-ce. So I don't want to change name of original owner

Reply:
That makes sense, but it would be nice to get confirmation from @arx-8 that they're okay being a maintainer for these types.

Reply:
@amcasey what.should I do next?

Reply:
arx-8 appears to have been inactive since May.  Since they can easily remove themselves in a future PR, it seems fine to go ahead.